trie/trieStorageManager.go

```diff
@@ -600,20 +614,33 @@ func isActiveDB(stsm *snapshotTrieStorageManager) bool {
 	return false
 }
 
-func isTrieSynced(stsm *snapshotTrieStorageManager) bool {
-	val, err := stsm.GetFromCurrentEpoch([]byte(common.TrieSyncedKey))
+func isTrieSynced(storer pruningStorer, epoch uint32) bool {
+	isTrieSyncedThisEpoch := isTrieSyncedInEpoch(storer, epoch)
+	isTrieSyncedLastEpoch := false
+
+	if epoch > 0 {
+		isTrieSyncedLastEpoch = isTrieSyncedInEpoch(storer, epoch-1)
+	}
+
+	isTrieSynced := isTrieSyncedThisEpoch || isTrieSyncedLastEpoch
+	log.Debug("isTrieSynced ", "value", isTrieSynced)
```

Comment:
let's print also `isTrieSyncedThisEpoch` and `isTrieSyncedLastEpoch`

Reply:
Done.